Mathlib/NumberTheory/Ostrowski.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2024 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Kurniadi Angdinata, Fabrizio Barroero, Laura Capuano, Nirvana Coppola,
+María Inés de Frutos Fernández, Sam van Gool, Silvain Rideau-Kikuchi, Amos Turchet,
+Francesco Veneziano
+-/
+
+import Mathlib.Analysis.Normed.Field.Basic
+import Mathlib.Analysis.SpecialFunctions.Pow.Real
+import Mathlib.Analysis.Normed.Ring.Seminorm
+
+/-!
+# Ostrowski’s Theorem
+
+The goal of this file is to prove Ostrowski’s Theorem which gives a list of all the nontrivial
+absolute values on a number field up to equivalence. (TODO)
+
+## References
+* https://kconrad.math.uconn.edu/blurbs/gradnumthy/ostrowskiQ.pdf
+* https://kconrad.math.uconn.edu/blurbs/gradnumthy/ostrowskinumbfield.pdf
+* J. W. S. Cassels, "Local fields", Cambridge University Press, 1986 https://doi.org/10.1017/CBO9781139171885
```

Comment:
For the three references, can you please follow the instructions here
https://leanprover-community.github.io/contribute/doc.html#citing-other-works ?

Reply:
Do you want to check before I merge?

Reply:
The bibtool thing doesn't work for me, so I'm not sure I did the correct thing.

Reply:
I think you have to put the item in alphabetical order *manually*

Reply:
Feel free to complain on zulip :P

Reply:
Happy to do things manually as long as I know that's what I'm supposed to do:)